kaal-presentation/src/main/kotlin/cz/eman/kaal/presentation/util/DiffCallbacks.kt

```diff
@@ -19,7 +19,7 @@ import androidx.recyclerview.widget.DiffUtil
  *
  * @since 0.8.0
  */
-class DiffCallback<in T>(
+class DiffCallback<in T: Any>(
```

Comment:
Because the method `areItemsTheSame` is not allowing nullable items anymore. The definition is:
`public abstract boolean areItemsTheSame(@NonNull T oldItem, @NonNull T newItem);`
One option is this as I have done, the other is do not allow oldItems and newItems to be nullable as
```
    private val oldItems: List<T & Any>,
    private val newItems: List<T & Any>,
```
    
  I don't know which solution is better but one of these is must.

Reply:
OK 👍